frame/transaction-payment/rpc/src/lib.rs

```diff
@@ -48,6 +48,20 @@ pub trait TransactionPaymentApi<BlockHash, ResponseType> {
 		encoded_xt: Bytes,
 		at: Option<BlockHash>,
 	) -> RpcResult<FeeDetails<NumberOrHex>>;
+
+	#[method(name = "payment_queryCallInfo")]
+	fn query_call_info(
+		&self,
+		encoded_call: Bytes,
+		at: Option<BlockHash>,
+	) -> RpcResult<RuntimeDispatchInfo<NumberOrHex>>;
+
+	#[method(name = "payment_queryWeightToFee")]
+	fn query_weight_to_fee(
```

Comment:
Should we have these rpc calls as addition to runtime api accessible via state.call endpoint.
Or the idea to stop adding the rpc calls at all.

Reply:
No. Don't add new RPCs please. Even full nodes have `state_call`.